module/VuFind/tests/unit-tests/src/VuFindTest/Section/AbstractSectionTestCase.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/**
+ * Abstract base class for navigation plugin tests.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see
+ * <https://www.gnu.org/licenses/>.
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+
+namespace VuFindTest\Section;
+
+use VuFind\Auth\ILSAuthenticator;
+use VuFind\Auth\Manager;
+use VuFind\Config\AccountCapabilities;
+use VuFind\Config\YamlReader;
+use VuFind\ILS\Connection;
+use VuFind\Navigation\AbstractMenu;
+use VuFind\Navigation\AccountMenu;
+use VuFind\Navigation\AdminMenu;
+use VuFind\Navigation\NavigationInterface;
+use VuFind\Navigation\PluginManager as NavigationManager;
+use VuFind\Section\PluginManager as SectionManager;
+use VuFind\Section\SectionInterface;
+use VuFind\Section\SectionService;
+use VuFind\Section\SectionServiceInterface;
+use VuFindTest\Container\MockContainer;
+use VuFindTest\Feature\ConfigRelatedServicesTrait;
+
+/**
+ * Abstract base class for navigation plugin tests.
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+abstract class AbstractSectionTestCase extends \PHPUnit\Framework\TestCase
+{
+    use ConfigRelatedServicesTrait;
+
+    /**
+     * Get a container with section related services.
+     *
+     * @return MockContainer
+     */
+    protected function getContainerWithSectionRelatedServices(): MockContainer
+    {
+        $container = new MockContainer($this);
+        $this->addSectionRelatedServicesToContainer($container);
+        return $container;
+    }
+
+    /**
+     * Add section related services to a mock container.
+     *
+     * @param MockContainer $container Mock container
+     *
+     * @return void
+     */
+    protected function addSectionRelatedServicesToContainer(MockContainer $container)
+    {
+        $container->set(YamlReader::class, new YAMLReader($this->getPathResolver()));
```

Comment:
Inconsistent case here for some reason:

```suggestion
        $container->set(YamlReader::class, new YamlReader($this->getPathResolver()));
```